docker-compose/docker-compose-certify/config/certify-default.properties

```diff
@@ -45,23 +45,24 @@ mosip.certify.security.ignore-auth-urls=**/actuator/**,**/error,**/swagger-ui/**
 
 ## ------------------------------------------ Discovery openid-configuration -------------------------------------------
 mosip.certify.discovery.issuer-id=${mosipbox.public.url}${server.servlet.path}
+mosip.certify.authorization.url=http://localhost:8088
 
 ##--------------change this later---------------------------------
 mosip.certify.supported.jwt-proof-alg={'RS256','PS256'}
 
-##  ---------------------------------------------- VCI ------------------------------------------------------------------
 
 ##----- These are reference to the oauth resource server providing jwk----------------------------------##
-mosip.certify.identifier=http://localhost:8088
-mosip.certify.authn.filter-urls={ '${server.servlet.path}/issuance/credential' }
-mosip.certify.authn.issuer-uri=http://localhost:8088/v1/esignet
+mosip.certify.cnonce-expire-seconds=40
+mosip.certify.identifier=${mosipbox.public.url}
```

Comment:
This is not getting used

Reply:
1. mosip.certify.identifier is being used as audience in proof validator [here](https://github.com/mosip/inji-certify/blob/cc7d66fd10bbd7289c0639ea3cceccc0581e191a/certify-service/src/main/java/io/mosip/certify/proof/JwtProofValidator.java#L48)
2. conunce expiry is used [here](https://github.com/mosip/inji-certify/blob/cc7d66fd10bbd7289c0639ea3cceccc0581e191a/certify-service/src/main/java/io/mosip/certify/services/VCIssuanceServiceImpl.java#L55)

Reply:
Then same should also be used in "credential_issuer", otherwise there is possibility for misconfiguration

Reply:
for I will merge for now, lets take care of this later

Reply:
> Then same should also be used in "credential_issuer", otherwise there is possibility for misconfiguration

yeah.